docs/TuxActions.md

```diff
@@ -299,21 +306,22 @@ Note that, as discussed [previously](#actionCategory-register), actionCategories
     actionCategory.add({text: 'new todo'});
 ```
 
+## <a id="Actions-Usage-Guidelines"></a>Actions Usage Guidelines [#](#Actions-Usage-Guidelines)
+When using `Actions` we highly recommend finding the categories of data your app will interact with and creating an `actionCategory` for each data type. Sometimes actionCategories will not overlap exactly with stores and that is totally acceptable.
+
+The real key with actionCategories is to take advantage of the `before` method to wrap up all of the business logic associated with an action verb so that it is totally encapsulated. This produces an extremely clean division of concerns wherein your views simply dispatch events (not knowing anything about what work that entails), your stores simply get the data from events (not know anything about what work went into generating that data), and your actions simply do the work of gathering and dispatching data (not knowing anything about what triggered the action or who needs the data).  The following example helps to demonstrate how `Actions` can be used to great effect in an app.
+
 ## <a id="TuxActions-Complete-Example"></a>TuxActions Complete Example [#](#TuxActions-Complete-Example)
 Taking everything we have learned let's look at a more realistic example of using TuxActions.
 
 ```
     var Actions = require('tux/Actions');
-
-    var messageActions = Actions.createActionCategory({
-      category: 'messages',
-      source: 'message components',
-      actions: ['get', 'create', 'delete']
-    });
+    var Stores = require('tux/Stores');
 
     var messageStore = Stores.createStore({
       _messages: [],
 
+      //the messageStores callbacks for the corresponding actions
```

Comment:
It might have been fixed with another issue from earlier in the doc. There's no issue now.